source/ios/AdaptiveCards/ADCIOSFramework/ADCIOSFramework/ACRHostConfig.h

```diff
@@ -0,0 +1,31 @@
+//
+//  ADCIOSAdaptiveHostConfig.h
+//  ADCIOSFramework
+//
+//  Created by jwoo on 6/7/17.
+//  Copyright © 2017 Microsoft. All rights reserved.
+//
+
+#import <Foundation/Foundation.h>
+#import <UIKit/UIKit.h>
+#import "TextBlock.h"
+#import "Image.h"
+#import "HostConfig.h"
+
+using namespace AdaptiveCards;
+
+@interface ACRHostConfig:NSObject
+
+- (CGSize) getImageSize:(std::shared_ptr<Image>) img;
```

Comment:
is it still true for shared pointer? I thought I was only copying pointers.

---
In reply to: [122550418](https://github.com/Microsoft/AdaptiveCards/pull/423#discussion_r122550418) [](ancestors = 122550418)